tests/testcases.py

```diff
@@ -188,6 +189,14 @@ def assertFileContents(self, file_path, contents):
     def assertFileContentsInContainer(self, service_name, file_path, contents):
         self.assertEqual(self.exec_in_container(service_name, 'cat {}'.format(file_path)), contents)
 
+    def assertFileNotInContainer(self, service_name, file_path):
+        self.assertEqual(self.exec_in_container(service_name,
+            'sh -c \'[ -f {} ] && echo "yes" || echo "no"\''.format(file_path)).rstrip(), 'no')
+
+    def assertFileInContainer(self, service_name, file_path):
+        self.assertEqual(self.exec_in_container(service_name,
+            'sh -c \'[ -f {} ] && echo "yes" || echo "no"\''.format(file_path)).rstrip(), 'yes')
```

Comment:
These should share code between each other, see some of the other examples in the class.